django_project/frontend/utils/metrics.py

```diff
@@ -0,0 +1,37 @@
+from collections import Counter
+
+from django.db.models import Sum
+from property.models import Property
+
+
+def calculate_population_categories(queryset):
```

Comment:
Hi @ketankartoza, could you please add a docstring here? It's a bit confusing to understand what the queryset is. Also, could you provide a type hint for the queryset?